actions/terra-k8s-linter/policy/deployments/naming_convention.rego

```diff
@@ -0,0 +1,22 @@
+package main
+
+import data.kubernetes
+
+
+deny_proper_service_account_name[msg] {
+  input.kind == "Deployment"
+  input.metadata.annotations["bio.terra.linter/serviceAccountName_exception"] != "enabled"
+  app_name := input.spec.template.spec.name
+  exp_service_account_name := sprintf("%s%s", [app_name, "-sa"])
```

Comment:
what's the `exp` in `exp_service_account_name` mean? otherwise I like the descriptive variable names!

Reply:
exp is commonly used in place of expected